pkg/controllers/tunnel/tunnel.go

```diff
@@ -88,23 +88,20 @@ func (t *TunnelController) processNextEndpoint() bool {
 		klog.V(4).Infof("Processing endpoint object: %v", obj)
```

Comment:
I've seen that pattern used in Go for defers. It could be a named function. Since the inline function makes up the bulk of the outer function, I don't have a strong preference either way.